Java/parkservices/src/main/java/com/amazon/randomcutforest/extendedrandomcutforest/threshold/ThresholdedRandomCutForestMapper.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amazon.randomcutforest.extendedrandomcutforest.threshold;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import com.amazon.randomcutforest.RandomCutForest;
+import com.amazon.randomcutforest.extendedrandomcutforest.threshold.state.BasicThresholderMapper;
+import com.amazon.randomcutforest.state.IStateMapper;
+import com.amazon.randomcutforest.state.RandomCutForestMapper;
+
+@Getter
+@Setter
+public class ThresholdedRandomCutForestMapper
+        implements IStateMapper<ThresholdedRandomCutForest, ThresholdedRandomCutForestState> {
+
+    @Override
+    public ThresholdedRandomCutForest toModel(ThresholdedRandomCutForestState state, long seed) {
+
+        RandomCutForestMapper randomCutForestMapper = new RandomCutForestMapper();
+        BasicThresholderMapper thresholderMapper = new BasicThresholderMapper();
+
+        RandomCutForest forest = randomCutForestMapper.toModel(state.getForestState());
+        BasicThresholder thresholder = thresholderMapper.toModel(state.getThresholderState());
+        return new ThresholdedRandomCutForest(forest, thresholder, state);
```

Comment:
This is a pattern we were avoiding in core: passing the state object itself to the model constructor. There may be pros and cons to this approach (e.g., if we pass the state to a constructor then we may end up in a situation where part of the mapping happens in this method and part happens in the constructor, which makes it harder to see what's happening), but I think we should be consistent with core.

Reply:
Done. Lombok and getters/setters made it easy; the long list moves from constructor to the mapper.